graphql2/schema.graphql

```diff
@@ -972,6 +973,11 @@ input UpdateUserInput {
   statusUpdateContactMethodID: ID
 }
 
+input SetUserRoleInput {
+  id: ID!
+  role: UserRole!
+}
```

Comment:
From `GraphQL` we can probably stick with `updateUser` and we can switch off of the `role` field in the backend. Keep `SetUserRole` in the `user.Store` though